athena-google-bigquery/src/main/java/com/amazonaws/athena/connectors/google/bigquery/BigQueryUtils.java

```diff
@@ -89,6 +89,17 @@ public static Credentials getCredentialsFromSecretsManager(java.util.Map<String,
                         "https://www.googleapis.com/auth/drive"));
```

Comment:
Please take the strings in line 88, 89, 99, 100 and make them into constants either in this file or in a util file.